### PR TITLE
Remove deprecated validator parameters

### PR DESCRIPTION
**Pull request by terencechain**
Open this on behave of @mcdee 

Reverts prysmaticlabs/prysm#5276

### CODE REVIEW

**Comment by prestonvanloon** [comment]
Doesn't this require adding documentation?

**Comment by terencechain** [comment]
cc @mcdee to answer any questions

**Comment by prestonvanloon** [comment]
Why was this reverted in the first place?

File: validator/node/node.go [hunk 1]
```diff
@@ -209,23 +212,6 @@ func selectKeyManager(ctx *cli.Context) (keymanager.KeyManager, error) {
 		opts = string(fileopts)
 	}
 
-	if manager == "" {
```
Comment (rauljordan) [inline validator/node/node.go:212]: Can we actually keep this in? This would be helpful

Comment (mcdee) [reply validator/node/node.go:212]: The deprecated vars have gone away so there's nothing for this code to do.

Would you like me to set a default for the keymanager?

**Pull request merged**